### ISSUE TITLE
Passed `registry` loses its prototype

### ISSUE THREAD

**Issue by SimenB** [SkeLLLa/fastify-metrics#9]
After this line

https://github.com/SkeLLLa/fastify-metrics/blob/9aa6453d8e3de787ad41c57eb5d8bf69b98ce666/src/index.ts#L107

```ts
opts.histogram.registers[0] instanceof client.Registry // true
extendedOpts.histogram.registers[0] instanceof client.Registry // false
```

This leads to 

```
TypeError: registryInstance.registerMetric is not a function
    at /Users/simen/repos/monorepo/node_modules/prom-client/lib/histogram.js:77:26
    at Array.forEach (<anonymous>)
    at new Histogram (/Users/simen/repos/monorepo/node_modules/prom-client/lib/histogram.js:73:20)
    at fastifyMetrics (/Users/simen/repos/monorepo/node_modules/fastify-metrics/dist/index.js:64:27)
    at Plugin.exec (/Users/simen/repos/monorepo/node_modules/avvio/plugin.js:129:17)
    at Boot.loadPlugin (/Users/simen/repos/monorepo/node_modules/avvio/plugin.js:263:10)
    at Task.release (/Users/simen/repos/monorepo/node_modules/fastq/queue.js:140:16)
    at worked (/Users/simen/repos/monorepo/node_modules/fastq/queue.js:182:10)
    at /Users/simen/repos/monorepo/node_modules/avvio/plugin.js:266:7
    at done (/Users/simen/repos/monorepo/node_modules/avvio/plugin.js:198:5)
    at check (/Users/simen/repos/monorepo/node_modules/avvio/plugin.js:222:9)
    at internal/process/task_queues.js:153:7
    at AsyncResource.runInAsyncScope (async_hooks.js:186:9)
    at AsyncResource.runMicrotask (internal/process/task_queues.js:150:8)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by SkeLLLa** [comment]
Should be fixed in 6.0.0-rc.4

**Comment by SkeLLLa** [comment]
Released v6 version, should be fixed in it.
